website/docs/blog.md

```diff
@@ -89,11 +89,11 @@ Or this.
 
 ## Feed
 
-You can generate RSS/ Atom feed by passing feedOptions.
+You can generate RSS/ Atom feed by passing feedOptions. By default, RSS and Atom feeds are generated.
```

Comment:
As we now generate feeds by default, can you document how to disable the feed generation?